src/commonMain/kotlin/com/algolia/search/model/response/ResponseLogs.kt

```diff
@@ -98,5 +109,13 @@ public data class ResponseLogs(
 
         public val nbApiCalls: Long
             get() = nbApiCallsOrNull!!
+
+        @Serializable
+        public data class InnerQuery(
```

Comment:
Open question here: In the codebase, (If I am not mistaken) the new data classes were generally declared in separate files. What's your preference here?

Reply:
The class `Log` itself is declared inside the `ResponseLogs` class. I've just followed the same pattern. However, in general I prefer to extract classes when it makes sense (more likely for reutilisability)